dto/scenario_testrun.go

```diff
@@ -9,22 +9,33 @@ import (
 type CreateScenarioTestRunBody struct {
 	ScenarioIterationId string `json:"scenario_iteration_id"`
 	ScenarioId          string `json:"scenario_id"`
+	RefIterationId      string `json:"ref_iteration_id"`
+	PhantomIterationId  string `json:"phantom_iteration_id"`
+	StartDate           string `json:"start_date"`
+	EndDate             string `json:"end_date"`
 	Period              string `json:"period"`
 }
 
 type ScenarioTestRunResp struct {
-	ScenarioId string    `json:"scenario_id"`
-	CreatedAt  time.Time `json:"created_at"`
-	ExpiresAt  time.Time `json:"expires_at"`
-	Status     string    `json:"status"`
+	Id              string `json:"id"`
+	ScenarioId      string `json:"scenario_id"`
+	RefIterationId  string `json:"ref_iteration_id"`
+	TestIterationId string `json:"test_iteration_id"`
+	StartDate       string `json:"start_date"`
```

Comment:
Hmm there is one thing which I don't exactly understand here, using time.Time for created_at/end_at seemed fine to me. 
Is there a specific goal behind it ?

Reply:
it concerns the dto that will be sent back to the front.. we could live it with a time type but string seemed to be simpler